server/index.js

```diff
@@ -4,30 +4,50 @@ const cors = require('cors');
 const app = express();
 const port = 5000;
 const models = require('./models');
+const passportSetup = require('./config/passport-setup');
 const userRoutes = require('./routes/userRoutes');
 const groupRoutes = require('./routes/groupRoutes');
 let db;
 
-app.use(cors());
+const corsOptions = {
+  origin: ['http://localhost:3000', 'http://localhost:5000'],
+  optionsSuccessStatus: 200, // some legacy browsers (IE11, various SmartTVs) choke on 204
+  credentials: true
+};
+
+const passport = require('passport');
+const authRoutes = require('./routes/auth-routes');
+const session = require('express-session');
+
+app.set('view engine', 'ejs');
+app.use(cors(corsOptions));
+app.use(require('cookie-parser')());
 app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({ extended: true }));
 
+app.use(session({secret: 'some_secret_key', resave: false, saveUninitialized: false }));
+app.use(passport.initialize());
+app.use(passport.session());
+
+app.use('/auth', authRoutes);
```

Comment:
can we prepend this with /api as well

Reply:
@harshyyy21 will need to update his google API settings but yes.

Reply:
updated